src/functionalTest/resources/1111006.metadata.json

```diff
@@ -9,14 +9,14 @@
   "envelope_classification": "exception",
   "scannable_items": [
       {
-        "document_control_number": "1111006",
+        "document_control_number": "$$1111006$$",
```

Comment:
is the middle part needed? maybe we could just provided the entire DCN?

Reply:
kept same formatting as before. dollar sign is not great for regex really. did not run integration suite in the office to verify everything as they take too long. will do that from home - much faster